pkg/collect/logs.go

```diff
@@ -107,10 +101,7 @@ func savePodLogs(ctx context.Context, bundlePath string, client *kubernetes.Clie
 
 	setLogLimits(&podLogOpts, limits, convertMaxAgeToTime)
 
-	fileKey := fmt.Sprintf("%s/%s", name, pod.Name)
-	if container != "" {
-		fileKey = fmt.Sprintf("%s/%s/%s", name, pod.Name, container)
-	}
+	fileKey := fmt.Sprintf("%s/%s/%s/%s", "cluster-resources/pods/logs", pod.Namespace, pod.Name, container)
```

Comment:
need to duplicate here, have one that is the 'old' path and one that's the new.  Then symlink the new one to the old one when it's saved.

Reply:
If I test this non-refactored change with `sbctl` we also miss the `*json` files located in `cluster-resources/`

I'm currently looking into adding the functionality for a symlink of the new location to the old location of the logs as described earlier and see if the other issue with missing `json` files is also resolved.